stumpy/config.py

```diff
@@ -14,3 +14,5 @@
 STUMPY_MAX_P_NORM_DISTANCE = np.finfo(np.float64).max
 STUMPY_MAX_DISTANCE = np.sqrt(STUMPY_MAX_P_NORM_DISTANCE)
 STUMPY_EXCL_ZONE_DENOM = 4
+STUMPY_TILE_SIZE = 4096
+STUMPY_DIAGONALS_CHUNK_SIZE = 100
```

Comment:
> Why 100? Why not also a factor of 2?

Good point, I forgot to experiment with this value so it's still what I had originally set. I can do that.

> Instead of `STUMPY_DIAGONALS_CHUNK_SIZE` I feel like this should be `STUMPY_N_DIAGONALS`. Otherwise, it sounds like we are chunking the diagonals. But instead, IIUC, this is the number of diagonals to traverse together?

That's correct, this represents the number of diagonals we are traversing together. I can change this to `STUMPY_N_DIAGONALS`.

Reply:
Yes, please